File: Java/Codigos Java/src/Fibonacci.java
public class Fibonacci{
    public static void main(String[] args) throws Exception {
    Scanner ler = new Scanner(System.in);
    int aux1 = 0;
    int aux2 = 1;
    int aux3;
    System.out.println("Digite quantos termos voce quer mostrar: ");
    aux1 = ler.nextInt();
    
    System.out.println(aux1);
    System.out.println(aux2);

    int contador = 3;
    
    while (contador <= num){
        aux3 = aux1 +aux2;
        System.out.println(aux3);
        aux1 = aux2;
        aux2 = aux3;
        contador +=1;
    };
}

File: JavaScript/Fibonacci.js
var num = parseInt(console.log("Digite quantos termos voce quer mostrar:"))
var aux1 = 0;
var aux2 = 1;
var aux3;
console.log(aux1)
console.log(aux2)

var contador = 3;
while (contador <= num){
    aux3 = aux1 +aux2
    console.log(aux3)
    aux1 = aux2
    aux2 = aux3
    contador +=1;
};
